### PR TITLE
fixed #16873

### LINKED ISSUE #16873: ort_perf_view tool doesn't load any profiling data
### Describe the issue

I am trying to visualize my inferenced profiling data and load it with this tool but seems, doesn't load anything.  

### To reproduce

- Loading profiling data from the same directory or different with ort_perf_view tool
- Tested with profiling data from onnxruntime and browser doesn't load any statistics data. It just remains blank. 

### Urgency

_No response_

### Platform

Linux

### OS Version

Ubuntu 22.04

### ONNX Runtime Installation

pip installation

### ONNX Runtime Version or Commit ID

1.15.0

### ONNX Runtime API

Python

### Architecture

X64

### Execution Provider

CUDA

### Execution Provider Library Version

_No response_

### PR DESCRIPTION
**Pull request by Red-Eyed**
### Description

Changed version of `anychart-ui` from `8.11.0` to `8.11.1`

### Motivation and Context
 Fixes #16873

### CODE REVIEW

**Review by hariharans29 — APPROVED** [review]

**Comment by hariharans29** [comment]
Can you please re-base with `main` ?

**Comment by baijumeswani** [comment]
[Red-Eyed](https://github.com/Red-Eyed) thank you for your contribution.

File: tools/perf_view/ort_perf_view.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <script type="text/javascript" src="https://cdn.anychart.com/releases/v8/js/anychart-core.min.js"></script>
     <script src="https://cdn.anychart.com/releases/8.11.0/js/anychart-ui.min.js"></script>
```
Comment (anhnami) [inline tools/perf_view/ort_perf_view.html:6]: The error remains due to this line. It should be 8.11.1 too.

**Pull request merged**